.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules/
+npm-debug.*
+*.jks
+*.p12
+*.key
+*.mobileprovision
```

Comment:
Additionally, might we want to .ignore the stuff that comes along with `sonarlint`? I know there's a couple of folders involved.

Reply:
We will when we get to that point!